### ISSUE TITLE
Add ability to check for 2nd update

### ISSUE THREAD

**Issue by rpappalax** [rpappalax/shavar-e2etest#3]
Current test checks for first update, but TP will mark the list as 'stale' after 45 minutes has passed and turn TP off on browser.
A good enhancement would be to run test as is then sleep 45 minutes, then check the TP is still enabled.
Note:
whether or not a data update has been pushed is irrelevant.  The fact that shavar 'checked-in' with a no-update, is still considered an update and will allow TP to remain enabled.

**Comment by monicachew** [comment]
> TP will mark the list as 'stale' after 45 minutes has passed and turn TP off on browser.

I told you a lie here: this is the desired end state but waiting on https://github.com/mozilla-services/shavar/issues/32.

**Comment by rpappalax** [comment]
Ah ok.  By 'desired end state,' do you mean that TP would shut off (at least until mozilla-services/shavar#32 gets fixed)?

**Comment by monicachew** [comment]
No, I mean that TP won't shut off after 45 minutes until mozilla-services/shavar#32 gets fixed. But the test will still work as proposed.

**Comment by rpappalax** [comment]
got it! ;)
